quiz-app/src/assets/translations/hin.json

```diff
@@ -0,0 +1,1017 @@
+			{
+				"id": 5,
+				"title": "पाठ 3 - सेंसर और एक्चुएटर्स के साथ भौतिक दुनिया के साथ बातचीत करें: प्री-लेक्चर क्विज़",
+				"quiz": [
+					{
+						"questionText": "एक एलईडी एक सेंसर है:",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "सेंसर का उपयोग किया जाता है:",
+						"answerOptions": [
+							{
+								"answerText": "भौतिक दुनिया से डेटा इकट्ठा करें|",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "भौतिक दुनिया को नियंत्रित करें|",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "केवल तापमान की निगरानी करें।",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "एक्चुएटर्स का उपयोग किया जाता है:",
+						"answerOptions": [
+							{
+								"answerText": "भौतिक दुनिया से डेटा इकट्ठा करें।",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "भौतिक जगत पर नियंत्रण करें।",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "बीमा जोखिमों की गणना करने के लिए|",
+								"isCorrect": "false"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 6,
+				"title": "पाठ 3 - सेंसर और एक्चुएटर्स के साथ भौतिक दुनिया के साथ बातचीत करें: पोस्ट-लेक्चर क्विज",
+				"quiz": [
+					{
+						"questionText": "डिजिटल सेंसर इस रूप में डेटा भेजते हैं:",
+						"answerOptions": [
+							{
+								"answerText": "वोल्टेज रेंजस",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "केवल हाई और लो वोल्टेज",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "ईमेल",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "बटन दबाने पर कौन सा डिजिटल सिग्नल भेजा जाता है?",
+						"answerOptions": [
+							{
+								"answerText": "0",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "1",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "आप पल्स-विड्त मॉड्यूलेशन का उपयोग करके डिजिटल डिवाइस से एनालॉग एक्ट्यूएटर को नियंत्रित कर सकते हैं",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "false"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 7,
+				"title": "पाठ 4 - अपने डिवाइस को इंटरनेट से कनेक्ट करें: प्री लेक्चर प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "IoT उपकरणों को हमेशा काम करने के लिए इंटरनेट से कनेक्ट होने की आवश्यकता होती है:",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					},
+					{
+						"questionText": "IoT डिवाइस हमेशा HTTP पर संचार करते हैं, वेब ऐप्स और अन्य वेब API के समान:",
```

Comment:
"questionText": "IoT डिवाइस हमेशा HTTP पर संचार करते हैं, वेब ऐप्स और अन्य वेब API के समान ___:",

हैं is missing in ___.